privacyidea/api/lib/postpolicy.py

```diff
@@ -775,3 +775,63 @@ def construct_radius_response(request, response):
         return make_response('', return_code)
     else:
         return response
+
+
+def mangle_challenge_response(request, response):
+    """
+    This policy decorator is used in the AUTH scope to
+    decorate the /validate/check endpoint.
+    It can modify the contents of the response "detail"->"message"
+    to allow a better readibilty for a challenge response text.
+
+    :param request:
+    :param response:
+    :return:
+    """
+    content = json.loads(response.data)
+    policy_object = g.policy_object
+    user_obj = request.User
+
+    header_pol = policy_object.get_action_values(action=ACTION.CHALLENGETEXT_HEADER,
```

Comment:
What do you think about passing ``unique=True`` here? I think we can't handle the case that we get more than one result anyway.

Reply:
Will do

Reply:
done in a94cc884